README.md

```diff
@@ -91,21 +91,18 @@ A simple wrapper for the Jaeger exporter of [opentelemetry-java](https://github.
-| otel.config.max.attrs               | OTEL_CONFIG_MAX_ATTRS              | unlimited      | Maximum number of attributes per span.                                                                                                                                                                                                                                                                                                                                                                    |
-| otel.config.max.attr.length         | OTEL_CONFIG_MAX_ATTR_LENGTH        | unlimited      | Maximum length of string attribute value in characters. Longer values are truncated.                                                                                                                                                                                                                                                                                                                      |
-| otel.config.max.events              | OTEL_CONFIG_MAX_EVENTS             | `256`          | Maximum number of events per span.                                                                                                                                                                                                                                                                                                                                                                        |
-| otel.config.max.links               | OTEL_CONFIG_MAX_LINKS              | `256`          | Maximum number of links per span.                                                                                                                                                                                                                                                                                                                                                                         |
-| otel.config.max.event.attrs         | OTEL_CONFIG_MAX_EVENT_ATTRS        | unlimited      | Maximum number of attributes per event.                                                                                                                                                                                                                                                                                                                                                                   |
-| otel.config.max.link.attrs          | OTEL_CONFIG_MAX_LINK_ATTRS         | unlimited      | Maximum number of attributes per link.                                                                                                                                                                                                                                                                                                                                                                    |
```

Comment:
Max attr length, max event & link attributes are no longer configurable by property in the SDK so I've removed them.